src/shared/bootspec.c

```diff
@@ -1482,25 +1477,25 @@ static int print_cmdline(
                 if (!t2)
                         return log_oom();
 
-                printf("  ukiCmdline: %s\n", t2);
-                final_cmdline = TAKE_PTR(t2);
+                printf("      options: %s\n", t2);
+                combined_cmdline = TAKE_PTR(t2);
         }
 
-        FOREACH_ARRAY(addon, global_arr->items, global_arr->count) {
-                print_addon(addon, "globalAddon");
-                if (!strextend(&final_cmdline, " ", addon->cmdline))
+        FOREACH_ARRAY(addon, global_arr->items, global_arr->n_items) {
+                print_addon(addon, "globaladdon");
```

Comment:
maybe "global-addon", i.e. the dash-based syntax is closer to how we did things so far